scanners/dispatcher/dispatcher.py

```diff
@@ -1,179 +1,108 @@
-import os
+import ast
+import emoji
 import sys
 import logging
 import requests
-import jwt
-import threading
-from flask import Flask, request
+from starlette.applications import Starlette
+from starlette.routing import Route, Mount, WebSocketRoute
+from starlette.responses import PlainTextResponse
+from utils import *
 
 logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 
-app = Flask(__name__)
 
-TOKEN_KEY = os.getenv("TOKEN_KEY")
+def startup():
+    logging.info(emoji.emojize("ASGI server started :rocket:"))
 
-hosts = [
-    "http://https-scanner.tracker.svc.cluster.local",
-    "http://ssl-scanner.tracker.svc.cluster.local",
-    "http://dmarc-scanner.tracker.svc.cluster.local",
-]
 
-dkim_flagged_hosts = [
-    "http://dkim-scanner.tracker.svc.cluster.local",
-    "http://dmarc-scanner.tracker.svc.cluster.local",
-]
-
-manual_scan_hosts = [
-    "http://https-scanner-manual.tracker.svc.cluster.local",
-    "http://ssl-scanner-manual.tracker.svc.cluster.local",
-    "http://dmarc-scanner-manual.tracker.svc.cluster.local",
-]
-
-manual_scan_dkim_flagged_hosts = [
-    "http://dkim-scanner-manual.tracker.svc.cluster.local",
-    "http://dmarc-scanner-manual.tracker.svc.cluster.local",
-]
-
-
-@app.route("/receive", methods=["GET", "POST"])
-def receive():
-
-    payload = {}
-    dkim_flag = False
+def initiate(received_payload, scan_type):
 
     try:
-        decoded_payload = jwt.decode(
```

Comment:
Is the JWT gone forever, if so do you want to update the api to reflect the changes, or do you want me to ?

Reply:
Updated